.travis.yml

```diff
@@ -57,6 +57,6 @@ matrix:
       dist: trusty
       install: skip
       jdk: openjdk8
-      script: mvn test -Pcheck-test-suites -B -V
+      script: mvn test -Pcheck-test-suites,all-java,all-scala,scala -B -V
```

Comment:
Why do we need this change here?

Reply:
@Mmuzaf I've add a compatibility test for indexes and forget add it to suite. But the check was OK. So I've found that ignite-compatibility module is excluded from pom.xml by default. For me this module is important one and the check must cover it. So I've add "all-java" profile to the check. This profile enables the ignite-examples module that requires the "scala" profile to be enabled.